umbraco/Umbraco-CMS
# V14: Deny login screen access

# Notes
- In some scenarios, you could manually navigate to the login screen (`/umbraco/login`), where you were not really supposed to be able to do that:

1. When you are already logged in, you should not be able to see the login screen
2. When you are resetting a password, and already logged in
3. When you are invited, but already logged in

From a server perspective, we needed a way to tell the client about the logged in status of the user, thus we have added a property to the model that get's send with, with whether the user is logged in or not.

# How to test
- Log in the backoffice
- Manually navigate to `/umbraco/login`, if you set a breakpoint, you should be able to see the `UserIsAlreadyLoggedIn` property being set to true.
